code/game/objects/items/stacks/nanopaste.dm

```diff
@@ -37,20 +37,27 @@
 
 		if(S)
 			if(BP_IS_ROBOTIC(S))
+				if(!S.get_damage())
+					to_chat(user, SPAN_NOTICE("There is nothing to fix here."))
+					return
+				for(var/datum/wound/W in S.wounds)
+					if(W.internal)
+						return
+					if(amount <= 0)
+						break
+					if(!do_mob(user, M, W.damage/5) && use(1))
+						to_chat(user, SPAN_NOTICE("You must stand still to repair \the [S]."))
+						break
+					if(!use(1))
```

Comment:
Due to that line `use(1)` check at 48 should be ommited/included here like `if(!use(2))`